File: GeradorDados/Models/TipoRisco.cs
namespace GeradorDados.Models;

public class TipoRisco
{
    public TipoRisco(int tiporiscoid, string descricao)
    {
        Tiporiscoid = tiporiscoid;
        Descricao = descricao;
    }

    public int Tiporiscoid { get; set; }
    public string Descricao { get; set; }
}

File: api/src/main/java/br/ucsal/clinica/model/TipoRisco.java
package br.ucsal.clinica.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "tipo_risco")
@AllArgsConstructor
@NoArgsConstructor
@Data
public class TipoRisco {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String descricao;
}
